app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,412 @@
+@import 'bootstrap';
+* {
```

Comment:
do not write css in this file. Use the application.scss for imports only

Reply:
modified accordingly. ✓